### PR TITLE
docs: Updated Roadmap document

### PR DESCRIPTION
**Pull request by Golodhros**
### Summary of Changes
Updates our roadmap document to remove already done items
Updates planned features and focus (biased by Lyft's roadmap)

### Documentation
Updates the doc

### CheckList

Make sure you have checked **all** steps below to ensure a timely review.

- [ ] PR title addresses the issue accurately and concisely, including a title prefix.
- [ ] PR includes a summary of changes.
- [ ] My commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -1,107 +1,112 @@
...
-*Status*: tech spec WIP
+## Short Term - Our Current focus
 
 #### Native lineage integration
-*What*: We want to create a native lineage integration in Amundsen, to better surface how data assets interact with each other.
 
-*Status*: tech spec out
+_What_: We want to create a native lineage integration in Amundsen, to better surface how data assets interact with each other.
+
+_Status_: ready to start
```
Comment (markgrover) [inline docs/roadmap.md:19]: I'd suggest "designs complete" here.

Comment (Golodhros) [reply docs/roadmap.md:19]: Sounds good!

File: docs/roadmap.md [hunk 2]
```diff
@@ -1,107 +1,112 @@
...
 #### Commonly joined tables / browsing the data model
-*What*: As a data user, I would like to see commonly joined tables and how to join them.
+
+_What_: As a data user, I would like to see commonly joined tables and how to join them.
 One option would be to show commonly joined tables and showing example join queries. Another option would be to provide a navigational experience for data model, showing foreign keys and which tables they come from.
 
-*Status*: planning has not started 
+_Status_: planning has not started
 
 #### Curated navigation experience
```
[the remark this answers is not in the document]
Comment (Golodhros) [reply docs/roadmap.md:54]: Makes sense!

File: docs/roadmap.md [hunk 3]
```diff
@@ -1,107 +1,112 @@
...
 
-*Status*: implementation has started (around 80% complete)  
+#### Push ingest API
 
-#### GET Rest API
-*What*: enable users to access our data map programmatically through a Rest API  
+_What_: We want to create a push API so that it is as easy as possible for a new data resource type to be ingested
 
-*Status*: implementation has started  
+_Status_: not planned
```
[the remark this answers is not in the document]
Comment (Golodhros) [reply docs/roadmap.md:64]: Yeah, that was an error, we don't know how it happened

**Comment by feng-tao** [comment]
@Golodhros  I will take a look later today as well, but I think we should include some of the ongoing RFCs in https://github.com/amundsen-io/rfcs.

File: docs/roadmap.md [hunk 4]
```diff
@@ -1,107 +1,112 @@
...
+> _Centralize a comprehensive and actionable map of all our data resources that can be leveraged to solve a growing number of use cases and workflows_
 
-*Status*: tech spec WIP
+## Short Term - Our Current focus
 
 #### Native lineage integration
-*What*: We want to create a native lineage integration in Amundsen, to better surface how data assets interact with each other.
 
-*Status*: tech spec out
+_What_: We want to create a native lineage integration in Amundsen, to better surface how data assets interact with each other.
```
Comment (feng-tao) [inline docs/roadmap.md:17]: if we are only targeting table level lineage in short term, we may want to make it explict.

File: docs/roadmap.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (markgrover) [reply docs/roadmap.md:64]: Hmm, that doesn't sound right though. For example, you can extract records from Kafka today which makes it a push model. Unless perhaps different people mean different things by 'push model'. Thoughts?
cc @feng-tao

File: docs/roadmap.md (hunk 1 above)
Comment (feng-tao) [reply docs/roadmap.md]: UX design complete?

File: docs/roadmap.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (feng-tao) [reply docs/roadmap.md:64]: I don't think we make the internal push model POC to open source. But I think with https://github.com/amundsen-io/amundsendatabuilder/blob/master/databuilder/extractor/kafka_source_extractor.py, as long as client could produce record based on certain schema, you could use that extractor to fetch record from kafka and persist back.

File: docs/roadmap.md (hunk 4 above)
Comment (Golodhros) [reply docs/roadmap.md:17]: I think we want both, and we have designs for both

**Review by devtagare — APPROVED** [review]

**Review by allisonsuarez — APPROVED** [review]
thanks!

**Comment by feng-tao** [comment]
@Golodhros @allisonsuarez  I don't think we include the one from @amundsen-io/rfcs ?

**Comment by Golodhros** [comment]
> @Golodhros @allisonsuarez I don't think we include the one from @amundsen-io/rfcs ?

Which one? This https://github.com/amundsen-io/rfcs/blob/master/rfcs/002-databuilder-record-abstraction.md ?

**Comment by markgrover** [comment]
Thanks! While we are at it, should we include a minor update to the push API?
Say, something like:

`Possible through [Kafka extractor](https://github.com/amundsen-io/amundsendatabuilder/blob/master/databuilder/extractor/kafka_source_extractor.py), though message format is not well defined. More work not planned`.

Flexible here, thoughts?

**Comment by Golodhros** [comment]
> [amundsen-io/rfcs#10](https://github.com/amundsen-io/rfcs/pull/10) and [amundsen-io/rfcs#13](https://github.com/amundsen-io/rfcs/pull/13), but let me create a sperate PR to include. Thanks @Golodhros !

Makes sense!

**Comment by Golodhros** [comment]
> Thanks! While we are at it, should we include a minor update to the push API?
> Say, something like:
> 
> `Possible through [Kafka extractor](https://github.com/amundsen-io/amundsendatabuilder/blob/master/databuilder/extractor/kafka_source_extractor.py), though message format is not well defined. More work not planned`.
> 
> Flexible here, thoughts?

Could you include this in your PR @feng-tao ?

**Comment by feng-tao** [comment]
sure

**Pull request merged**